Source/Timeline/EventView.swift

```diff
@@ -58,9 +58,11 @@ open class EventView: UIView {
   }
 
   func updateWithDescriptor(event: EventDescriptor) {
+    print (event.font)
```

Comment:
This `print` statement should be removed before the pull request could be merged.